src/components/sdk/introduction.stories.mdx

```diff
@@ -2,23 +2,26 @@
 
 # SDK Components
 
-Components for building out a realtime meeting experience. They have a dependency on providers being rendered in your application tree, which subsequently have a dependency on the `amazon-chime-sdk-js`.
+SDK Components provide you the capability to build out a realtime meeting experience easily. They have a dependency on providers being rendered in your application tree, which subsequently have a dependency on the `amazon-chime-sdk-js`.
```

Comment:
SDK Components provide you the capability to easily build out a realtime meeting experience.